packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/sift-science.js

```diff
@@ -47,9 +47,13 @@ class SiftScience extends Component {
     const helperDomain = path(['domains', 'walletHelper'], walletOptions)
     const sfoxSiftScience = path(['platforms', 'web', 'sfox', 'config', 'siftScience'], walletOptions)
 
-    let url = `${helperDomain}/wallet-helper/sift-science/#/key/${sfoxSiftScience}/user/${userId}`
+    let url = `${helperDomain}/wallet-helper/sift-science/#/key/${sfoxSiftScience}/user/${userId.getOrElse()}`
     url += tradeId ? `/trade/${tradeId}` : ''
 
+    if (!userId.getOrElse(null)) {
+      return null
+    }
+
```

Comment:
`const id = userId.getOrElse(null)`
`if (!userId) return`

Then you don't need 53-56 and change `userId` in line 50 to `id`